api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -464,7 +466,9 @@ def serialize_venue(cls, venue: offerers_models.Venue) -> dict:
             "tags": [criterion.name for criterion in venue.criteria],
             "banner_url": venue.bannerUrl,
             "_geoloc": position(venue),
-            "is_eligible_for_strict_search": offerers_api.is_venue_eligible_for_strict_search(venue),
+            # TODO: remove the first field when no longer used
```

Comment:
micro-détail : 

```suggestion
            # TODO: remove "is_eligible_for_strict_search" key when the app does not use it anymore
```

C'est surtout pour préciser **par qui** c'est utilisé (au cas où le lecteur pressé ne cherche que dans le backend !).